extensions/amp-analytics/0.1/test/test-iframe-transport.js

```diff
@@ -135,52 +136,6 @@ describes.realWin('amp-analytics.iframe-transport', {amp: true}, env => {
     expect(createPerformanceObserverSpy).to.be.called;
   });
 
-
-  it('logs poor performance of vendor iframe', () => {
```

Comment:
Not deleted, just reordered and then changed a quite a bit.
The main change is the value of frameUrl2, and that we're no longer setting srcdoc. That (and some other deleted stuff) was hoop-jumping necessitated by doNotLoadExternalResourcesInTest() in testing/iframe.js, which is no longer needed because of the new allowExternalResources: true on line 155.